CMakeLists.txt: Fix detection of Python

Signed-off-by: Stephen Rothery <steve@cryptoguide.tips>

--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -4,7 +4,7 @@ include_directories (${CMAKE_CURRENT_SOURCE_DIR})
 if (CMAKE_VERSION VERSION_LESS 3.24)
   find_package (Python 3.6 COMPONENTS Interpreter Development)
 else ()
-  find_package (Python 3.6 COMPONENTS Interpreter Development.Module)
+  find_package (Python 3.6 COMPONENTS Interpreter)
 endif ()
 
 if (Python_FOUND)
